theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,24 @@
+<nav id="main-navigation">
+  <% function recNav(navig) { %>
+    <ul> 
+      <% for ({title, link, sub_items} of navig) { %>
+        <li>
+            <a href="<%= link %>"><%= title %></a>
+              <% if (sub_items) { %>
+                <a class="has-children" href="<%= link %>"></a>
```

Comment:
As mentioned in https://github.com/browser-automation/cba-website/pull/7#discussion_r409176587 this still looks like a markup hack to visually show arrow element, though previously(with span) it has fewer impact on accessibility.